### ISSUE TITLE
Zoom

### ISSUE THREAD

**Issue by StevanWhite** [murraycu/android-galaxyzoo#41]
Feature suggestion:

Image zoom.  The image doesn't need to translate, only zoom on on the central object.
On a small device, the central object might be only a few mm across!

**Comment by murraycu** [comment]
Yes, this would be really good to have. I tried this a while ago, but I should try again:
https://github.com/murraycu/android-galaxyzoo/commit/39c70bf52dc66acf8fa5cfb36bde74ea614bf998
